### ISSUE TITLE
Support Segment timeline for type dynamic

### ISSUE THREAD

**Issue by AkamaiDASH** [Dash-Industry-Forum/Conformance-Software#159]
Conformance test completed
Segment timeline for type dynamic is not supported, only MPD will be tested.

**Comment by waqarz** [comment]
@AkamaiDASH can you point to the MPD?

**Comment by waqarz** [comment]
Might be related to https://github.com/Dash-Industry-Forum/Conformance-Software/issues/160

**Comment by joywelt** [comment]
@AkamaiDASH Could you please provide the MPD URL which you used for testing? 
I tested with http://vm2.dashif.org/livesim-dev/segtimeline_1/testpic_6s/Manifest.mpd, which is dynamic and has SegmentTimeline attibute. The conformance software checked the MPD in this case.

**Comment by AkamaiDASH** [comment]
I was out on Holiday just getting back.  I will look but pretty sure I can not share publicly.  Ill follow up soon.

**Comment by AkamaiDASH** [comment]
It was for a manifest with SegmentTemplate with SegTimeline inside of seg temp node.

**Comment by waqarz** [comment]
Does the issue persist?

**Comment by AkamaiDASH** [comment]
This is an example stream that displays the message "Segment timeline for type dynamic is not supported, only MPD will be tested."
https://svt01-a.akamaized.net/u/svt01.mpd

**Comment by waqarz** [comment]
Yes, this features is currently not supported; we bring this up in the test or IOP discussion. Thanks for pointing.
